vault/dendron.release.2021-11-02.md

```diff
@@ -0,0 +1,74 @@
+---
+id: 4q0nOxVnDvAMD18zyNebk
+title: 'Release Notes (version 0.66)'
+desc: ''
+updated: 1635872354186
+created: 1635870362508
+---
+
+Dendron 0.66 has sprouted  🌱
+
+An early version of task notes is now available to test out! Take a look and feel free to provide feedback on [RFC 13 - Todo Notes in the GitHub Discussion](https://github.com/dendronhq/dendron/discussions/1358).
+
+This release also includes some breaking changes due to some renaming of commands, which can be referrenced in the changelog.
+
+### Highlights
+- feat(notes): task notes (create modifier & editor highlighting) ([[docs|dendron.topic.tasks]]) (#1583) @kaan
```

Comment:
for the release notes, remove the github pr and user tags. we only keep them in the changelog